.github/rsi-schema.json

```diff
@@ -1,185 +1,244 @@
-            {
-               "name":"basic-directions",
-               "directions":4
-            },
-            {
-               "name":"basic-delays",
-               "delays":[
-                  [
-                     0.1,
-                     0.1
-                  ]
-               ]
+            "states": [
+                {
+                    "name": "basic",
+                    "license": "CC-BY-SA-3.0",
+                    "copyright": "Taken from CODEBASE at COMMIT LINK"
+                },
+                {
+                    "name": "basic-directions",
+                    "license": "CC-BY-SA-3.0",
+                    "copyright": "Taken from CODEBASE at COMMIT LINK",
+                    "directions": 4
+                },
+                {
+                    "name": "basic-delays",
+                    "license": "CC-BY-SA-3.0",
+                    "copyright": "Taken from CODEBASE at COMMIT LINK",
+                    "delays": [
+                        [
+                            0.1,
+                            0.1
+                        ]
+                    ]
+                },
+                {
+                    "name": "basic-delays-directions",
+                    "license": "CC-BY-SA-3.0",
+                    "copyright": "Taken from CODEBASE at COMMIT LINK",
+                    "directions": 4,
+                    "delays": [
+                        [
+                            0.1,
+                            0.1
+                        ],
+                        [
+                            0.1,
+                            0.1
+                        ],
+                        [
+                            0.1,
+                            0.1
+                        ],
+                        [
+                            0.1,
+                            0.1
+                        ]
+                    ]
+                }
+            ]
+        }
+    ],
+    "required": [
+        "version",
+        "size",
+        "states"
+    ],
+    "title": "RSI Schema",
+    "type": "object",
+    "properties": {
+        "version": {
+            "$id": "#/properties/version",
+            "default": "",
+            "description": "RSI version integer.",
+            "title": "The version schema",
+            "type": "integer"
+        },
+        "license": {
+            "$id": "#/properties/license",
+            "default": "",
+            "description": "The license for the associated icon states. Restricted to SS14-compatible asset licenses.",
+            "deprecated": true,
+            "deprecationMessage": "Whole-file licencing is not recommended. You should be doing individual licencing on each state!",
+            "enum": [
+                "CC-BY-SA-3.0",
+                "CC-BY-SA-4.0",
+                "CC-BY-NC-3.0",
+                "CC-BY-NC-4.0",
+                "CC-BY-NC-SA-3.0",
+                "CC-BY-NC-SA-4.0",
+                "CC0-1.0"
```

Comment:
Missing `CC-BY-4.0` and `CC-BY-3.0` from

https://github.com/space-wizards/RobustToolbox/blob/76b46479b6a1ad1d960fba7c8df878469590d1ea/Schemas/rga_validators.py#L6-L18

Also `MIT` and `Custom`. However applicable that is to *image files*. But never hurts to have around.